site/content/docs/cicd/deployment-settings-file/overview.md

```diff
@@ -0,0 +1,53 @@
+# Creating a deployment settings file
+
+A deployment settings file allows you to define the deployment settings of your application in a _JSON_ format. This JSON file can be version controlled and plugged into your CI/CD system for future deployments. 
+
+Deployment settings files could be used as a way to automate your deployments or use them in a [CI/CD setting](../cicd.md) where you would define all the settings that you need to apply for your deployment and then use the `--apply` flag on the CLI to link to the deployment setting file. 
+
+By doing this, the AWS .NET deployment tool reads all the settings you have defined and applies them to the deployment. You will need to do a final confirmation to initiate the deployment in the CLI. However, you can bypass the confirmation by making the deployment a silent one. This can be done by adding the `--silent` flag which will turn off any prompts.
+
+The deployment settings file has the following structure:
+
+```json
+{
+      "AWSProfile": <AWS_PROFILE>,
+      "AWSRegion": <AWS_REGION>,
+      "ApplicationName": <APPLICATION_NAME>,
+      "RecipeId": <RECIPE_ID>,
+      "Settings": <JSON_BLOB>
+}
+```
+
+* _**AWSProfile**_: The name of the AWS profile that was used during deployment.
```

Comment:
Call out for this and the AWSRegion and ApplicationName properties are optional and that these values can be overridden by using the appropriate command line switch. That way a user can see that they can craft a file that could potential be used for multiple profiles and regions.

Reply:
Done